action.yml

```diff
@@ -29,14 +38,14 @@ inputs:
   imageToBuild:
     description: |
       'The custom name of the image that is to be built, pushed to ACR and deployed to the Container App by this action.
```

Comment:
Similarly, should we replace this other occurrence of "to ACR" by "to the container registry"> Similar comment below for the imageToDeploy parameter.